py-polars/polars/expr/datetime.py

```diff
@@ -859,6 +859,37 @@ def time(self) -> Expr:
         -------
         Expr
             Expression of data type :class:`Time`.
+
+        Examples
+        --------
+        >>> from datetime import datetime
+        >>> df = pl.DataFrame(
+        ...     {
+        ...         "datetime": pl.datetime_range(
+        ...             datetime(2020, 1, 1),
+        ...             datetime(2020, 1, 2),
+        ...             "5ms",
+        ...             eager=True,
+        ...         ),
+        ...     }
+        ... )
+        >>> df.with_columns(pl.col("datetime").dt.time().alias("time"))
+        shape: (17_280_001, 2)
```

Comment:
can we keep docstring examples small please? 5 rows should be enough, multiple examples with millions of rows will slow down CI

Reply:
If I can add to this: personally I think it's clearer to define 3 `datetime` objects and use those rather than use date range everywhere.

If you pick three datetimes that are sufficiently different, you can probably reuse them across all the examples and clearly show what each of them do.

Reply:
OK, that makes sense. I made the examples I added smaller and have tried to reuse them more. I also shrunk a few of the larger examples that were already there.